### PR TITLE
README update

### PR DESCRIPTION
**Pull request by JAndrassy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,11 +18,9 @@ The firmware does not (and likely will not) implement the whole set of AT comman
 
 The major differences are:
 
-~~1. Only the station mode (AT+CWMODE=1) is supported, no AP or mixed mode.~~
```
Comment (JiriBilek) [inline README.md:21]: Are you sure all the AT+CWMODEs are supported? I think we support only WIFI_STA (1) and WIFI_AP (2), we don't support the mixed mode WIFI_AP_STA

**Review by JiriBilek — CHANGES_REQUESTED** [review]
Thanks

File: README.md (hunk 1 above)
Comment (JAndrassy) [reply README.md:21]: mixed mode is supported

Comment (JAndrassy) [reply README.md:21]: AT+CWMODE is mapped directly to WiFi.mode for all 3 values

Comment (JiriBilek) [reply README.md:21]: Ok, I misunderstood the code in cmd_AT_CWMODE().

**Pull request merged**